test/utils/base.py

```diff
@@ -3,13 +3,27 @@
 # SPDX-License-Identifier: MPL-2.0
 
 import unittest
+from unittest.mock import patch
 
 import numpy as np
 import pandas as pd
 
 
 class BaseTestCase(unittest.TestCase):
 
+    def setUp(self) -> None:
+        super().setUp()
+        self.patchers = []
+        self.patchers.append(patch("ktpbase.log.logging.get_logger", create=True))
+        self.patchers.append(patch("ktpbase.config.config.ConfigManager.get_instance", create=True))
+        for patcher in self.patchers:
+            patcher.start()
+
+    def tearDown(self) -> None:
+        super().tearDown()
+        for patcher in self.patchers:
+            patcher.stop()
+
     # TODO for some reason this does not work : has not attribute addTypeEqualitty
     # def __init__(self, *args, **kwargs):
```

Comment:
Can we remove this commented out code?

Reply:
I used an old depreciated method, should have used `addTypeEqualityFunc()`  I have fixes this now so no commented code anymore, good point! See 5b287377ee04b280600dbc491ed44f044ecdf4e3.